pages/premium.php

```diff
@@ -108,13 +110,13 @@ function premium(): bool {
 	if(isset($_POST['user_reset'])) {
 		try {
             if($player->team) {
-                throw new RuntimeException("You must leave your team before resetting!");
+                throw new Exception("You must leave your team before resetting!");
```

Comment:
```suggestion
                throw new RuntimeException("You must leave your team before resetting!");
```
This looks like bad merge conflict, these should all be RuntimeException now